osctiny/utils/conf.py

```diff
@@ -157,7 +157,11 @@ def get_credentials(url: typing.Optional[str] = None) \
 
     if sshkey is not None:
         if not sshkey.exists():
-            raise ValueError(f"SSH key from config does not exist: {sshkey}")
+            # if it is just a key file name, look at the default SSH dir (which is the most
+            # common case)
+            sshkey = Path("~").expanduser() / ".ssh" / sshkey
```

Comment:
Let's make this just a tiny bit more compact :wink: 

```suggestion
            sshkey = Path.home() / ".ssh" / sshkey
```